src/api/java/com/minecolonies/api/colony/ICitizenData.java

```diff
@@ -480,4 +480,17 @@
      * @param justAte true if justAte, false to reset.
      */
     void setJustAte(boolean justAte);
+
+    /**
+     * Drain experience from the worker.
+     * @param maxDrain the max to drain.
+     * @return the drained amount x2.
+     */
+    double drainExperience(int maxDrain);
```

Comment:
Why x2? Either rename this, or change the logic. x2 does not make sense.

Reply:
Yeah it's now by a config value. Will change text